forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -3655,10 +3655,8 @@ public static void addStaticAbility(final KeywordInterface inst, final CardState
             String effect = "Mode$ CantTransform | ValidCard$ Creature.Self | ExceptCause$ SpellAbility.Daybound | Secondary$ True | Description$ This permanent can't be transformed except by its daybound ability.";
             inst.addStaticAbility(StaticAbility.create(effect, state.getCard(), state, intrinsic));
         } else if (keyword.equals("Decayed")) {
-            String effect = "Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't block. | " +
-                    "Secondary$ True";
+            String effect = "Mode$ CantBlockBy | ValidBlocker$ Creature.Self | Secondary$ True | Description$ CARDNAME can't block.";
             StaticAbility st = StaticAbility.create(effect, state.getCard(), state, intrinsic);
-            st.setSVar("SacrificeEndCombat", "True");
```

Comment:
This does seem wrong like this (sorry, think it was me) but shouldn't it be kept somehow for the CreatureEvaluator?

Something like:
`state.getCard().setSVar` ?

Reply:
It can just check for the KW in that case, I'll add it later

Reply:
@tool4ever there are some cards that uses SacrificeEndCombat without other keywords, i want to make it look at the trigger directly without need for extra checks or svars